setup.py

```diff
@@ -36,5 +41,6 @@
     py_modules=[module.stem for module in here.glob("src/*.py")],
```

Comment:
since we are not going to use `pathlib`, this line should be changed.
```suggestion
    py_modules=["epiweeks"],
```